editor/editor_inspector.cpp

```diff
@@ -3397,9 +3315,15 @@ void EditorInspector::update_tree() {
 				ep->connect("multiple_properties_changed", callable_mp(this, &EditorInspector::_multiple_properties_changed));
 				ep->connect("resource_selected", callable_mp(this, &EditorInspector::_resource_selected), CONNECT_DEFERRED);
 				ep->connect("object_id_selected", callable_mp(this, &EditorInspector::_object_id_selected), CONNECT_DEFERRED);
-				// `|` separator used in `make_help_bit()` for formatting.
-				ep->set_tooltip_text(property_prefix + p.name + "|" + doc_info.description);
-				ep->set_doc_path(doc_info.path);
+
+				// `|` separator used in `EditorHelpTooltip` for formatting.
+				if (theme_item_name.is_empty()) {
+					ep->set_tooltip_text("property|" + classname + "|" + property_prefix + p.name + "|");
+				} else {
+					ep->set_tooltip_text("theme_item|" + classname + "|" + theme_item_name + "|");
+				}
```

Comment:
I think this needs to be guarded by `use_doc_hints`. First of all, because that disables the feature, and second, because without `use_doc_hints` class name may be incorrect.